### ISSUE TITLE
Breaking function calls into multiple lines silently breaks builds

### ISSUE THREAD

**Issue by dejaime** [godotengine/godot#8013]
My system: http://pasted.co/ca4970b9
(with kernel 4.8)
Using Godot 2.1 Stable

If you break a function call into multiple calls it works flawlessly in the editor, with no runtime problems or syntax errors whatsoever.
```
	tween.interpolate_property(
		self,
		"transform/pos",
		get_pos(),
		ev.pos,
		0.1,
		Tween.TRANS_LINEAR,
		Tween.EASE_IN_OUT)
```

Still, if one tries to export the project with such a call, the export (apparently) completes successfully.

But as soon as one tries to execute such build, it fails to execute (crashes completely) or simply renders that specific script broken.

![image](https://cloud.githubusercontent.com/assets/2046959/23829075/227d590c-06c5-11e7-84b3-f6df3617cc1b.png)

Calling functions without breaking them in several lines (or using the \ character to do so) works fine on both the editor and the exported builds.

[Issue.zip](https://github.com/godotengine/godot/files/836296/Issue.zip)

**Comment by bojidar-bg** [comment]
Seems like I broke something with #6694...

**Comment by bojidar-bg** [comment]
@dejaime Actually, can you confirm you are using the export templates for the right version (since that's a common problem around here)?

**Comment by Hinsbart** [comment]
@dejaime Yeah, it seems like a problem with your templates.
I just tested this on 2.1.2 with official 2.1.2 templates. It works fine both in the editor and exported.

**Comment by dejaime** [comment]
Indeed, It is possible that I am using outdated templates, as I tried (without any success) to fix the Sprite3D bug (#7964).

I'll verify this as soon as I have a chance.

Thanks you

**Comment by dejaime** [comment]
Yes, apparently, rebuilding the templates and editor has solved the issue.
Stupid me forgot to take note of the commit I was using though.

Looks like I messed up, sorry about that!
